layouts/page/pricing.html

```diff
@@ -2,17 +2,19 @@
     <header class="header-hero">
         <div class="container mx-auto text-center flex-col max-w-2xl">
             <h1>Pulumi for every team and individual.</h1>
-            <p>
-                Whether you're using open source, our free Community Edition, or using
-                Pulumi in your team or entire organization, we have a solution for you.
+            <p class="text-gray-400 mb-8">
+                Try any edition risk-free for 14-days. No credit card required.
             </p>
+            <a data-track="team-starter" class="text-center btn btn-lg btn-blue mx-auto mt-8 py-4 px-6" href="https://app.pulumi.com/site/organizations/add">
```

Comment:
```suggestion
            <a data-track="hero-free-trial" class="text-center btn btn-lg btn-blue mx-auto mt-8 py-4 px-6" href="https://app.pulumi.com/site/organizations/add">
```

The `data-track` for this should describe what it is so we can call it out in Google Analytics a bit more.